lib/sequenceserver/links.rb

```diff
@@ -116,25 +116,25 @@ def uniprot
       }
     end
 
-    def svg_download
-      {
-        :order => 2,
-        :title => 'SVG',
-        :url   => '#',
-        :class => 'export-to-svg',
-        :icon  => 'fa-download'
-      }
-    end
-
-    def png_download
-      {
-        :order => 2,
-        :title => 'PNG',
-        :url   => '#',
-        :class => 'export-to-png',
-        :icon  => 'fa-download'
-      }
-    end
+    # def svg_download
```

Comment:
Please remove the code, instead of commenting it out. If this code is required again, we can fetch it from git.